packages/tools/eslintBabylonPlugin/package.json

```
@@ -24,5 +24,6 @@
         "@microsoft/tsdoc-config": "~0.16.1"
     },
     "devDependencies": {
+        "@types/eslint": "^9.6.1"

```

nit - we are using eslint 8.X, better to have this:

```suggestion
        "@types/eslint": "^8.56.10"
```